sparkbox/bouncy-ball
# Simplify javascript architecture

## Description

Major changes include:

- Migrate postcss to plain css (and remove the compile step).
- Migrate custom code from using browserify + common-js to ES modules (and remove the bundle step).
- Use Snowpack to install ES-module-compatible 3rd party dependencies.
- Update custom third-party dependencies (PrismJS & Modernizr).
- Update linting tooling.

Fixes #77

## Testing instructions:

1. Pull down the branch and run `npm ci` to install dependencies
2. Start the server: `npm start`
3. Browse to the server url and walk through the functionality to ensure everything is working as expected.
    - Demos load and work as expected.
    - The "docs pane" opens and works as expected.
    - The url hash is updated for each demo (browsing directly to a link with the hash will load the corresponding demo).
